src/lib/hyrise.cpp

```diff
@@ -1,8 +1,16 @@
 #include "hyrise.hpp"
 
+#include <jemalloc/jemalloc.h>
+
 namespace opossum {
 
 Hyrise::Hyrise() {
+  // Work around for a performance regression in jemalloc:
```

Comment:
*Workaround

Reply:
Still WIP, no point in reviewing this yet. This part will have to go, anyway.